src/app/executions/tasks-runtime/tasks-runtime.component.ts

```diff
@@ -92,6 +105,18 @@ export class TasksRuntimeComponent implements OnInit {
                 preBarWidth = 99;
             }
             task.preBarWidth = preBarWidth;
+
+            const waitingBeforeBarWidth = (task.started_at_relative / workflowDurationSec) * 100;
```

Comment:
I would add some description to the doc string of the function, explaining the new elements you've added.

Reply:
done, but not sure if I did it correct